packages/transformers/src/modules/generic/http_get/http.get.transformer.ts

```
@@ -75,15 +75,20 @@ export class HttpGetTransformer implements ITransformer {
         })
-            .then(resp => {
-                if (!resp.ok) {
-                    this.telemetryLogger.sendErrorTelemetry(xmsg, `Request failed with code: ${resp.status}`);
-                    throw new Error(`Request failed with code: ${resp.status}`);
+        .then(resp => {
+            if (!resp.ok) {
+                this.telemetryLogger.sendErrorTelemetry(xmsg, `Query: ${this.config.url}${this.config.query ?? ''}; Headers: ${JSON.stringify(this.config.headers)}; Request failed with code: ${resp.status}`);
+                throw new Error(`Request failed with code: ${resp.status}`);
+            } else {
+                const contentType = resp.headers.get('content-type');
+                if (contentType && contentType.includes('application/json')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)